src/aleph/vm/controllers/firecracker/snapshot_manager.py

```diff
@@ -95,18 +95,18 @@ def run_snapshots(self) -> None:
         )
         job_thread.start()
 
-    async def start_for(self, execution: VmExecution, frequency: Optional[int] = None) -> None:
-        if not execution.is_instance:
+    async def start_for(self, vm: AlephFirecrackerExecutable, frequency: Optional[int] = None) -> None:
+        if not vm.is_instance:
             msg = "Snapshots are not implemented for programs."
             raise NotImplementedError(msg)
 
         default_frequency = frequency or settings.SNAPSHOT_FREQUENCY
 
-        vm_hash = execution.vm_hash
+        vm_hash = vm.vm_hash
         snapshot_execution = SnapshotExecution(
             scheduler=self._scheduler,
             vm_hash=vm_hash,
-            execution=execution,
+            execution=vm,
```

Comment:
Should this argument be renamed ?

Reply:
I don't think so, because it's the vm execution, also the class name is `AlephFirecrackerExecutable`, so I think that the name is already well-defined.